README.md

```diff
@@ -1,19 +1,24 @@
+<!-- START_METADATA
 ---
-title: Introduction
 sidebar_position: 1
 hide_table_of_contents: true
-pagination_prev: null
 pagination_next: null
+pagination_prev: null
 ---
+END_METADATA -->
+
+# Introduction
 
-# Webhooks API [BETA]
+💥 Work in progress 💥
 
-An API to register one or more callback urls per named event per sales unit.
+You can **subscribe** to published **event types** by registering webhooks and
+will then be **notified** when any matching events are published for payments
+that match the given sales unit id.
```

Comment:
```suggestion
that match the given Sales unit
```
Hm - what is the public term we use for this again 🙈 ? Halp @khenrichsen

Reply:
We usually say "sales unit"